std/math.d

```diff
@@ -4893,24 +4964,33 @@ private:
     {
         version (InlineAsm_X86_Any)
         {
-            version (Win64)
+            asm nothrow @nogc
             {
-                asm nothrow @nogc
-                {
-                    naked;
-                    mov     8[RSP],RCX;
-                    fclex;
-                    fldcw   8[RSP];
-                    ret;
-                }
```

Comment:
This deleted code looks very clever. Unfortunately, there is no comment on what it achieves. The generic variant below seems to work just fine on Win64, and it's much simpler. So I'm tearing down this fence without knowing why it was put up.